services/agents/mongodb.go

```diff
@@ -29,7 +29,7 @@ import (
 )
 
 // mongodbExporterConfig returns desired configuration of mongodb_exporter process.
-func mongodbExporterConfig(service *models.Service, exporter *models.Agent) *agentpb.SetStateRequest_AgentProcess {
+func mongodbExporterConfig(service *models.Service, exporter *models.Agent, debug debugValue) *agentpb.SetStateRequest_AgentProcess {
```

Comment:
This is just a matter of preference. Instead of passing a boolean parameter in each function call I would prefer to have a way to set a value for `enableDebug`